packages/volto/src/middleware/api.js

```diff
@@ -222,17 +222,40 @@ const apiMiddlewareFactory =
           }
 
           const { settings } = config;
-          if (getState().apierror.connectionRefused) {
+          const state = getState();
+          if (state.apierror.connectionRefused) {
             next({
               ...rest,
               type: RESET_APIERROR,
             });
           }
           if (type === GET_CONTENT) {
+            // automatically set redux userSession.token if the user logged in another tab
+            // if (!(subrequest || __SERVER__)) {
+            //   const cookies = new Cookies();
+            //   const token = cookies.get('auth_token');
+            //
+            //   if (token && !state.userSession?.token) {
+            //     dispatch({
+            //       type: 'LOGIN_SUCCESS',
+            //       result: {
+            //         token,
+            //       },
+            //     });
+            //   } else if (state.userSession?.token && !token) {
+            //     dispatch({
+            //       type: 'LOGOUT_SUCCESS',
+            //       result: {
+            //         token,
+            //       },
+            //     });
+            //   }
+            // }
+
```

Comment:
It's not clear why you would add commented code. Suggest removing all the above.